### PR TITLE
OX6-2: Remove Transaction Account

### PR DESCRIPTION
**Pull request by p1fb**

### CODE REVIEW

**Comment by andrefatchip** [comment]
Doing this would remove displaying the unwanted feature, yes. 
But it would also kill displaying TXID and Reference of that order and wether performing a capture nor a debit would work anymore.

So I would rather do the surgery than chopping the head off :-)

File: application/views/admin/tpl/fcpayone_order.tpl [hunk 1]
```diff
@@ -905,91 +905,10 @@
...
-                                                <strong>[{oxmultilang ident="FCPO_BALANCE"}]: <span [{if $stat->fcpotransactionstatus__fcpo_balance->value < 0}]style="color: red;"[{/if}]>[{$stat->fcpotransactionstatus__fcpo_balance->value|number_format:2:',':''}]&nbsp;[{$stat->fcpotransactionstatus__fcpo_currency->value}]</span></strong>
-                                            </td>
-                                        </tr>
-                                    </table>
-                                [{/if}]
-                            [{/foreach}]
-                    </div>
-                [{else}]
-                    Kein Transaktions-Status eingegangen.
-                [{/if}]
```
Comment (andrefatchip) [inline application/views/admin/tpl/fcpayone_order.tpl]: Just removing this part would do the trick. Furthermore I would recommend to check the controller for $stat which then would not be needed anymore and should be removed too.

Comment (andrefatchip) [reply application/views/admin/tpl/fcpayone_order.tpl]: Damn, I'm sorry. Thought you just wanted to kill the whole file. That's not the case and I approve your first attempt. Forget my nonsense above...

Comment (p1fb) [reply application/views/admin/tpl/fcpayone_order.tpl]: Thank you!

**Comment by hreinberger** [comment]
won't fix (for now)

**Pull request closed without merging**